processor/src/stack/tests.rs

```diff
@@ -403,7 +403,7 @@ fn build_helpers_partial(num_overflow: usize, next_overflow_addr: usize) -> Stac
 }
 
 /// Returns values in stack top columns of the provided trace at the specified row.
-fn read_stack_top(trace: &[Vec<Felt>; STACK_TRACE_WIDTH], row: usize) -> StackTopState {
+fn pop_stack_top(trace: &[Vec<Felt>; STACK_TRACE_WIDTH], row: usize) -> StackTopState {
     let mut result = [ZERO; STACK_TOP_SIZE];
     for (value, column) in result.iter_mut().zip(trace) {
         *value = column[row];
```

Comment:
Seems like this change wasn't rebased correctly.

Reply:
Indeed! renamed back to `read_stack_top`